qfdmo/forms.py

```diff
@@ -36,7 +36,72 @@
 )
 
 
-class AddressesForm(forms.Form):
+class MapForm(GetFormMixin, CarteConfigFormMixin, forms.Form):
+    """
+    This forms holds all the hidden input used to display the Acteur results in the
+    frontend.
```

Comment:
```suggestion
    This forms holds all the input used to display the Acteur results in the
    frontend.
    Most of them are hidden
```

- [ ] revoir commentaire